tests/network-tests/src/consts.ts

```diff
@@ -92,6 +92,7 @@ export const proposalTypeToProposalParamsKey = {
   'VetoProposal': 'vetoProposalProposalParameters',
   'UpdateGlobalNftLimit': 'updateGlobalNftLimitProposalParameters',
   'UpdateChannelPayouts': 'updateChannelPayoutsProposalParameters',
+  'FreezePallet': 'setInvitationCountProposalParameters',
```

Comment:
matching what was set in runtime, but I think `setInvitationCountProposalParameters` should be `freezePalletCountProposalParameters` or whatever name we end up choosing if the proposal name is changed.

Reply:
oh yeah you are right it was a mistake on my end...
I will fix it in the freeze proposal PR